.github/workflows/test.yml

```diff
@@ -13,14 +13,10 @@ on:
     branches: [ master ]
 
 env:
-  VAULT_ADDR: https://clotho.broadinstitute.org:8200
```

Comment:
Refactored the `test.yml` github action a bit. It no longer calls `local-dev/render-config.sh` to render secrets. Instead it relies on GitHub secrets populated by Terraform. See https://github.com/broadinstitute/terraform-ap-deployments/pull/1742

Reply:
I'll test this more thoroughly once the terraform PR is thumbed and applied